src/platforms/Points/usePointsCardProps.tsx

```diff
@@ -1,18 +1,22 @@
+import useDynamicRewardUserAmount from "platforms/Token/hooks/useDynamicRewardUserAmount"
 import Star from "static/icons/star.svg"
 import { GuildPlatformWithOptionalId, PlatformName } from "types"
 import { useRolePlatform } from "../../components/[guild]/RolePlatforms/components/RolePlatformProvider"
 
 const usePointsCardProps = (guildPlatform: GuildPlatformWithOptionalId) => {
   const rolePlatform = useRolePlatform()
   const { name, imageUrl } = guildPlatform.platformGuildData
+  const { dynamicUserAmount } = useDynamicRewardUserAmount(rolePlatform)
+
+  const score = !!rolePlatform?.dynamicAmount
+    ? dynamicUserAmount ?? "some"
+    : rolePlatform?.platformRoleData?.score
 
   return {
     type: "POINTS" as PlatformName,
     image: imageUrl || <Star />,
     // if undefined at admin setup -> "some", if saved with no value (empty string) -> 0
-    name: `Get ${(rolePlatform?.platformRoleData?.score ?? "some") || 0} ${
-      name || "points"
-    }`,
+    name: `Get ${score ?? 0} ${name || "points"}`,
```

Comment:
I think we should show the DynamicTag here too similarly as in PointsReward, cause now the card can be misleading for dynamic points (looks like it's static with the shown value set)

Reply:
The user's points are not necessarily calculated from a single dynamic reward, they are summed from all of the point rewards. The dynamic modal only displays the calculation for a single specific reward, it would be very complicated to display a calculation accross all of them here.